aviary/variable_info/variable_meta_data.py

```diff
@@ -1204,9 +1204,35 @@
                      "LEAPS1": None
                      },
     option=True,
+    units="lbm",
+    desc='required fuel reserves: directly in lbm',
+    default_value=0,
+)
+
+add_meta_data(
+    Aircraft.Design.RESERVES_FRACTION,
+    meta_data=_MetaData,
+    historical_name={"GASP": 'INGASP.FRESF2',  # the real name is INGASP.FRESF
```

Comment:
If the value in the historical name is not the real value, how will that affect our Fortran-to-Aviary converter? @jkirk5 ?

Reply:
My understanding is that historical_name creates a link between Aviary variable name and the one in gasp_default_values.dat. In the case of fuel reserves, 'INGASP.FRESF' is mapped to 'Aircraft.Design.RESERVES', 'INGASP.FRESF2' is mapped to 'Aircraft.Design.RESERVES_FRACTION'. I have to create 'INGASP.FRESF2' when I split 'Aircraft.Design.RESERVES'.

Reply:
No, fortran_to_aviary directly checks variable_metadata for name matching, so this won't work. The defaults files aren't used for conversion in the same way.

Because we have split this variable in Aviary, we need to add logic to fortran_to_aviary to handle getting the legacy value to where it goes. Check out line 479 on Fortran_to_Aviary.py for an example of how I handled this for the mass scaling factors in FLOPS. Have the original historical name map to reserves_fraction, then check if the value is actually total mass - if so, update RESERVES and set RESERVES_FRACTION to 1.0.

Never "invent" a historical name, this just adds confusion. Leave historical_name for RESERVES empty (or None).

Reply:
- I removed "invented" historical names.
- I use `Fortran_to_Aviary` to check and set `Aircraft.Design.FIXED_RESERVES_FUEL` and `Aircraft.Design.RESERVES_FRACTION`.